### DISCUSSION TITLE
Persistent file got corrupted

### DISCUSSION THREAD

**Discussion by chaoticfame** [Monika-After-Story/MonikaModDev#9148] [QA - tech support + general] [answered]
so hi guys, im new here i would just like to ask some help regarding my MAS files, so i resetted my pc and backed up the renpy folder and persistent files of my monika, but after putting them on my new pc, it says that the persistent i put is corrupted, any ways on how to fix this? Thanks

**Comment by Iceninja64** [comment, marked as answer]
Try to load another persistent file and it should work
